ltp-production.yaml

```diff
@@ -1757,3 +1757,13 @@ projects:
     url: https://bugs.linaro.org/show_bug.cgi?id=5976
     active: true
     intermittent: true
+  - environments: *environments_all
+    notes: >
+      shmget02 fails on compat mode 64-bit kernel booting on 32-bit userspace
+      shmget02.c:95: TFAIL: shmget(1644199826, 4278190080, 1536) expected EINVAL: ENOMEM (12)
+    projects: *projects_all
+    test_names:
+    - ltp-syscalls/shmget02
+    url: https://bugs.linaro.org/show_bug.cgi?id=5977
```

Comment:
reported upstream? url?

Reply:
Reported internally and did not get any response.

Reply:
Reported upstream.